app/controllers/lti_controller.rb

```diff
@@ -63,6 +76,30 @@ def init
 
     private
 
+    def find_remote_user_assignment(lti_email, orgs = @organization.self_and_descendants)
+
+        assignments = UserAssignment.joins(:user).where( {
+            :user_assignments => { :organization_id => orgs}, 
+            :users => { :email => lti_email }
+        } )
+        # could check for blank username here in the quary
+
+        return nil unless assignments.count == 1
+        assignments.first
+    end
+
+    def find_lti_user_by_eamil(user_email, orgs = @organization.self_and_descendants)
+        
+        user = User.joins(:user_assignments).where( {
+            :user_assignments => { :organization_id => orgs }, 
+            :users => { :email => user_email }
+        } )
```

Comment:
use `users` for query results like this. `user` by convention refers to a single record.